### PR TITLE
Release 2.54.0

### PR DESCRIPTION
**Pull request by pgrzesik**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
...
+- **Variables:** Unconditionally deprecate old vars engine extensions ([#9827](https://github.com/serverless/serverless/pull/9827)) ([b7f4e08](https://github.com/serverless/serverless/commit/b7f4e08661cd149a29ae7107241a16928dc606eb)) ([Mariusz Nowak](https://github.com/medikoo))
+
+### Maintenance Improvements
+
+- **AWS API Gateway:** Deprecate default for `identitySource` for `request` authorizers with disabled caching ([#9825](https://github.com/serverless/serverless/pull/9825)) ([0e01d9e](https://github.com/serverless/serverless/commit/0e01d9e337860b9d1136586fbfaf0c43ac21cde0)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- **CLI:** Deprecate `-v` as alias for `--verbose` ([#9811](https://github.com/serverless/serverless/commit/53b41eb53aeefe22dc29b785a428f3b184906d2c)) ([53b41eb](https://github.com/serverless/serverless/commit/53b41eb53aeefe22dc29b785a428f3b184906d2c)) ([Piotr Grzesik](https://github.com/pgrzesik))
```
Comment (medikoo) [inline CHANGELOG.md:21]: IMO both of this should be in features. They're technically step 1 of a breaking change

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,24 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.54.0](https://github.com/serverless/serverless/compare/v2.53.1...v2.54.0) (2021-08-12)
+
+### Features
+
+- **AWS IAM:** Resign from deprecating old format of IAM settings ([#9778](https://github.com/serverless/serverless/pull/9778)) ([847dd8f](https://github.com/serverless/serverless/commit/847dd8f47503ddf03971122e5700110aa3ff77d1)) ([Piotr Grzesik](https://github.com/pgrzesik))
```
[the remark this answers is not in the document]
Comment (pgrzesik) [reply CHANGELOG.md:9]: whoops :grimacing:

File: CHANGELOG.md (hunk 1 above)
Comment (pgrzesik) [reply CHANGELOG.md:21]: I didn't consider them a feature but rather a refactor, but I think it also make sense to have them in features section :+1:

Comment (medikoo) [reply CHANGELOG.md]: Theoretically (unlikely scenario) if someone is on `SLS_DEPRECATION_NOTIFICATION_MODE=error`, this change may introduce sudden crashes in deployments.

**Review by medikoo — APPROVED** [review]
👍

**Pull request merged**